zuul.d/jobs.yaml

```diff
@@ -205,6 +205,29 @@
         name: cloud_docs
         pass-to-parent: true
 
+- job:
+    name: promote-api-ref
+    parent: otc-promote-docs-base
+    description: |
+      Publish api-ref document to.
+      This job is run in a promote pipeline to publish documents built
+      in the gate pipeline.
+    # We only publish the master branch, so no need to run
+    # for changes on other branches.
+    branches:
+      - master
+      - main
```

Comment:
can we add here `devel`?

Reply:
in principle yes, but the issue is that we only can have single branch per project here, otherwise we overwrite docs. devel is something contradicting to "master/main".
Do we have already api-ref tox jobs in TF or somewhere else (except octavia and future docs projects)?

Reply:
In TF we have only tox releasenote job now.